tests/Unit/TestCreation.hpp

```diff
@@ -48,10 +48,12 @@ struct TestCreationOpt {
 /// \ingroup TestingFrameworkGroup
 /// Construct an object from a given string.
 ///
-/// The string must not contain the
-/// name of the option (which is the name of the struct by default if no
-/// `name()` function is present). For example, to create a struct with a
-/// `SizeT` named option the following would be used:
+/// The string must not contain the name of the option (specifically, the class
+/// being created from options, which is the name of the struct by default if
+/// no `name()` function is present). For example, to create a struct named
+/// `ClassWithoutMetavariables` with a `SizeT` named option the following would
```

Comment:
Should "SizeT named option" -> "option tag SizeT" or similar? (I think in the call earlier we agreed on wording at least)